report/test/renderer/details-renderer-test.js

```diff
@@ -173,6 +173,14 @@ describe('DetailsRenderer', () => {
           totalBytes: 71654,
           wastedBytes: 30470,
           wastedPercent: 42,
+          entity: 'example.com',
+        },
+        {
+          url: 'https://example2.com',
+          totalBytes: 7165,
+          wastedBytes: 3047,
+          wastedPercent: 42,
+          // missing entity classification
```

Comment:
better to do `entity: undefined` than a comment i think?

Reply:
yeah; agree